### PR TITLE
image: resolve abolute path from src

### PR DESCRIPTION
**Pull request by pared**
Related to iterative/dvc#7664

### CODE REVIEW

**Review by daavoo — COMMENTED** [review]
@pared sorry for my lack of understanding but I'm not sure I follow.

The HTML generated by this P.R. https://github.com/iterative/dvc/pull/7664 looks fine even without this P.R? 

HTML there contains absolute paths to the image so when I open the HTML in the browser it renders correctly?

**Comment by pared** [comment]
@daavoo Yes, it will work.
Hovewer, if we want to make HTML reports portable (eg copy dir and send via email) we need working relative paths. In iterative/dvc#7549 I made plots use relative paths, but relative to `cwd` and not `index.html`.

File: src/dvc_render/image.py [hunk 1]
```diff
@@ -20,17 +22,26 @@ class ImageRenderer(Renderer):
 
     EXTENSIONS = {".jpg", ".jpeg", ".gif", ".png"}
 
-    def partial_html(self) -> str:
+    def partial_html(self, output_dir=None, **kwargs) -> str:
```
Comment (daavoo) [inline src/dvc_render/image.py:25]: Do we need this arg? Afaik if `outs` was passed on DVC side, it should be present in `self.properties`:

https://github.com/iterative/dvc/blob/d0e05c4b66dc6e8b5e4ee46b622d42ab8b3fb01d/dvc/render/match.py#L48-L49

It makes more ense here if we expect this to be used outside DVC 🤔

Comment (pared) [reply src/dvc_render/image.py:25]: 🤦

Comment (pared) [reply src/dvc_render/image.py]: After thinking a bit about it, i think that this was a proper approach. `out` contains information where to write the images. So in a sense it contains the same information as `datapoints.src` just without exact name of the file. When generating HTML via `render_html`, we can provide `index.html` file path, and it is the one path we are interested in the most. To be sure that our relpath is correct, we need to evaluate `index.html` path and calculate relative path to its `dirname`. Renamed `output_dir` to more informative `html_path`.

**Pull request merged**